dz-eight/src/main/java/org/example/StudentsGroup.java

```diff
@@ -0,0 +1,53 @@
+package org.example;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class StudentsGroup {
+    private Student groupLead;
+    private List<Student> students;
+    private List<String> tasks;
+
```

Comment:
Add here and use `Map<String, List<Student>> compleatedTasks = new HashMap<>();` to mark that students compleated tasks